src/main/java/frc/team2412/robot/subsystem/MonitoringSubsystem.java

```diff
@@ -0,0 +1,29 @@
+package frc.team2412.robot.subsystem;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class MonitoringSubsystem extends SubsystemBase {
+    ShuffleboardTab tab = Shuffleboard.getTab("Power Monitoring");
+    NetworkTableEntry totalPower;
+    PowerDistribution powerDistributionPanel;
```

Comment:
These three variables should start as `private` - we can change later if needed.